cypress/e2e/MapPageSpec.cy.ts

```diff
@@ -0,0 +1,103 @@
+/// <reference types="cypress" />
```

Comment:
This is amazing, Ann, very thorough user flow!  Tomorrow I'll add some testing for the popups.